store/store_test.go

```diff
@@ -4648,6 +4660,7 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreDecorateOrdersTokenExpired(c *C) {
 	helloWorld := &snap.Info{}
 	helloWorld.SnapID = helloWorldSnapID
 	helloWorld.Prices = map[string]float64{"USD": 1.23}
+	helloWorld.Paid = true
```

Comment:
I must say that in all of the diff snippets above I was surprised we didn't just say:
```
helloWorld := &snap.Info{
   SnapID: ...
}
```